rock.osdeps

```diff
@@ -136,3 +138,10 @@ thin: gem
 grape: gem
 sprockets: gem
 
+qtruby:
+    default:
+        gem: qtbindings
+        osdep: [qt4, qt4-opengl, qt-designer, qt4-webkit]
```

Comment:
the designer is installing some dev-libraries needed for the vizkit ui loading mechanism